style/js/shield_defs.js

```diff
@@ -158,6 +158,21 @@ export function loadShields(shieldImages) {
     colorLighten: "#613214",
   };
 
+  shields["US:AZ"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_az_2,
+      shieldImages.shield40_us_az_3,
+    ],
+    textColor: "black",
+    padding: {
+      left: 4,
+      right: 3,
+      top: 3,
+      bottom: 4,
+    },
+  };
+  shields["US:AZ:Loop"] = shields["US:AZ"];
+
```

Comment:
```suggestion
  shields["US:AZ:Loop"] = banneredShield(shields["US:AZ"], ["LOOP"]);

```

Reply:
While I don't see it as strictly necessary, as the loop routes currently don't numerically overlap with other state routes, I don't see anything wrong with displaying the LOOP banner either.